Presentation/Home/HomeListView.swift

```diff
@@ -110,13 +113,166 @@ struct HomeListView: View {
                 .padding(.bottom, 20)
                 .padding(.leading, 30)
             
+            VStack(spacing: 0) {
+                HStack(spacing: 0) {
+                    Text("폴더")
+                        .reazyFont(.text1)
+                        .foregroundStyle(.gray700)
+                    
+                    Spacer()
+                    
+                    Button(action: {
+                        createFolder.toggle()
+                    }) {
+                        Image("newfolder")
+                            .renderingMode(.template)
+                            .resizable()
+                            .scaledToFit()
+                            .frame(height: 20)
+                            .foregroundStyle(.gray800)
+                    }
+                }
+                .padding(.bottom, 14)
+                
+                ScrollView {
+                    VStack(spacing: 0) {
+                        ForEach(rootFolders, id: \.id) { folder in
+                            FolderListCell(
+                                folder: folder,
+                                level: 0,
+                                expandedFolders: $expandedFolders,
+                                childFolders: childFolders(of:),
+                                toggleExpansion: toggleExpansion,
+                                hasChildren: hasChildren(folder:),
+                                selectedFolderID: $selectedFolderID
+                            )
+                            .padding(.top, 10)
+                        }
+                    }
+                }
+            }
+            .padding(.leading, 30)
+            .padding(.trailing, 16)
+            
             Spacer()
         }
         .padding(.top, 24)
         .background(.primary2)
     }
+    
+    private var rootFolders: [Folder] {
+        homeViewModel.folders.filter { $0.parentFolderID == nil }
+    }
+    
+    private func childFolders(of folderID: UUID?) -> [Folder] {
+        homeViewModel.folders.filter { $0.parentFolderID == folderID }
+    }
+    
+    private func hasChildren(folder: Folder) -> Bool {
+        !childFolders(of: folder.id).isEmpty
+    }
+    
+    private func toggleExpansion(_ folder: Folder) {
+        if expandedFolders.contains(folder.id) {
+            expandedFolders.remove(folder.id)
+        } else {
+            expandedFolders.insert(folder.id)
+        }
+    }
 }
 
+
+struct FolderListCell: View {
```

Comment:
Cell이 재활용이 될 경우 새로운 파일에 넣거나 아닌 경우 아래와 같이 private을 붙이는 것이 좋아 보입니다!!
```suggestion
private struct FolderListCell: View {
```